fishtank/src/node.ts

```diff
@@ -16,7 +16,11 @@ export class Node {
     this.backend = cluster['backend']
   }
 
+  private get containerName(): string {
+    return this.cluster['containerName'](this.name)
+  }
+
   remove(): Promise<void> {
-    return this.backend.remove([this.name], { force: true, volumes: true })
```

Comment:
Node class unit tests

Reply:
There are already unit tests for this method. Do you want more tests? If so, what do you want to be covered that is not already covered?

Reply:
https://github.com/iron-fish/ironfish-tank/blob/e07d629d02993c472c16f80eecfa55cd2c140874/fishtank/src/node.test.ts#L16
```
 const node = await cluster.spawn({ name: 'my-test-container' })
```
Can you verify the node name? since there is change in return value. I realize the best way is not in Node class but in Cluster class unit test, can you add expect for all the spawn() return and boostrap return ?